connector/failoverconnector/README.md

```diff
@@ -0,0 +1,79 @@
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+
+[in development]:https://github.com/open-telemetry/opentelemetry-collector#in-development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+
+## Supported Pipeline Types
+
+| [Exporter Pipeline Type] | [Receiver Pipeline Type] | [Stability Level] |
+| ------------------------ | ------------------------ |-------------------|
+| traces | traces | [In development]  |
+| metrics | metrics | [In development]           |
+| logs | logs | [In development]           |
+
+[Exporter Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#exporter-pipeline-type
+[Receiver Pipeline Type]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/connector/README.md#receiver-pipeline-type
+[Stability Level]: https://github.com/open-telemetry/opentelemetry-collector#stability-levels
+<!-- end autogenerated section -->
+
+Allows for health based routing between trace, metric, and log pipelines depending on the health of target downstream exporters.
+
+## Configuration
+
+If you are not already familiar with connectors, you may find it helpful to first visit the [Connectors README].
+
+The following settings are available:
+
+- `priority (required)`: list of pipeline level priorities in a 1 - n configuration, multiple pipelines can sit at a single priority level.
+- `retry_interval (required)`: the frequency at which the pipeline levels will attempt to reestablish connection with all higher priority levels
+- `retry_gap (required)`: the amount of time between trying two separate priority levels in a single retry_interval timeframe
+- `max_retry (required)`: the maximum retries per level
+
+Example:
+
+```yaml
+connectors:
+  failover:
+    priority:
+      - [traces/first, traces/fourth]
```

Comment:
Since these are at the same priority level, I wonder if it would be a bit clearer to new users if we renamed "fourth" to "also_first" or anything to that effect. It seems like the current name implies that it's somehow treated as a different priority.